app/facades/food_facade.rb

```diff
@@ -0,0 +1,11 @@
+class FoodFacade
+
+  def self.search(search)
+    info = FoodService.search(search)
+
+  end
```

Comment:
I think you'd get to this soon, but looks like your next step here should be converting this info into plain old ruby objects to send back to the controller.